single_tech_samples/synapseanalytics/sample3_integration_testing/infrastructure/modules/diagnostic_settings.bicep

```diff
@@ -0,0 +1,105 @@
+param project string
+param env string
+param deployment_id string
+param loganalytics_workspace_name string
+param synapse_workspace_name string
+param synapse_sql_pool_name string
+param synapse_spark_pool_name string
+
+var commonPrefix = '${project}-diag-${env}-${deployment_id}'
+
+resource synapseWorkspace 'Microsoft.Synapse/workspaces@2021-03-01' existing = {
+  name: synapse_workspace_name
+}
+
+resource synapseSqlPool001 'Microsoft.Synapse/workspaces/sqlPools@2021-03-01' existing = {
```

Comment:
this module has several resources appended with a number (1, 2, 001, etc.).

This is a sample, so that may make sense in this case, but I'm curious if there is a better name that could be used to delineate how each resource is used in the context of the sample. 

Please feel free to comment and resolve if there isn't a better name for this that adds value/awareness

Reply:
same comment as above. cc @devlace